dracut/30ignition/ignition-quench.service

```diff
@@ -16,4 +16,4 @@ After=sysroot-boot.service
 Type=oneshot
 # We will only run if GRUB detected this file. Fail if we are unable to
 # remove it, rather than risking rerunning Ignition at next boot.
-ExecStart=/bin/rm /sysroot/boot/flatcar/first_boot
+ExecStart=sh -c 'if [ ! -e /sysroot/boot/flatcar/first_boot ] && [ ! -e /sysroot/boot/coreos/first_boot ]; then echo "error: /sysroot/boot/(flatcar|coreos)/first_boot not found"; exit 1; else  [ -e /sysroot/boot/flatcar/first_boot ] && rm /sysroot/boot/flatcar/first_boot; [ -e /sysroot/boot/coreos/first_boot ] && rm /sysroot/boot/coreos/first_boot; fi'
```

Comment:
`s/sh -c/\/bin\/sh -c/`.

The shell script code inside `''` looks correct.
But I am not sure if it would be easy to maintain such a long line in `ExecStart`.
Could we maybe make it simpler by using `ConditionPathExists` or `AsssertPathExists`?

Reply:
I think `AssertPathExists` is not expressive enough and we would want to error if deletion fails for some reason.

Reply:
Thanks, the absolute path is used now.